test/index.js

```diff
@@ -1,5 +1,5 @@
-const test = require('blue-tape')
-const gradient = require('../build/index')
+import test from 'blue-tape'
```

Comment:
Not sure if we should use `ECMAScript Modules` inside test files since we usually run these files using Node without any transpiling tools

Reply:
Good, this should be merged then